platform/checkout/cisco-8000.ini

```diff
@@ -1,3 +1,3 @@
 [module]
 repo=git@github.com:Cisco-8000-sonic/platform-cisco-8000.git
-ref=v0.1
+ref=v0.6
```

Comment:
Where is your onie image? In the release artifacts?

Reply:
yes, onie will be provided as an artifact.